### PR TITLE
TC_14-002.08-AHeader > Search Box| Case insensitive search

### PR DESCRIPTION
**Pull request by leliks818**
https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/178

### CODE REVIEW

File: cypress/e2e/headerSearchBox.cy.js [hunk 1]
```diff
@@ -67,16 +65,25 @@ describe('US_14.002 | Header > Search Box', () => {
...
-    
     cy.get('input#search-box').type('no matches{Enter}');
-
     cy.get('.error').should('have.text', 'Nothing seems to match.');
     
   });
   
-});
+  it('TC_14.002-08-A |Case insensitive search', () => {
+      cy.get('div[class="login page-header__hyperlinks"] a[class="model-link"]').should('be.visible').click()
```
Comment (Rom-T) [inline cypress/e2e/headerSearchBox.cy.js:78]: I think it's better to use it like this `cy.get('.login .model-link')`

File: cypress/e2e/headerSearchBox.cy.js [hunk 2]
```diff
@@ -67,16 +65,25 @@ describe('US_14.002 | Header > Search Box', () => {
     cy.get("[name='Submit']").click();
     cy.get("#search-box").click();
     cy.get("#search-box").type("MA");
-    
-    cy.get(".yui-ac-bd").should('have.text', 'manage');
-  });
+    cy.get(".yui-ac-bd").should('have.text', 'manage')
+  })
```
Comment (Rom-T) [inline cypress/e2e/headerSearchBox.cy.js:69]: Why did you remove the semicolon?

File: cypress/e2e/headerSearchBox.cy.js [hunk 3]
```diff
@@ -67,16 +65,25 @@ describe('US_14.002 | Header > Search Box', () => {
...
-
     cy.get('.error').should('have.text', 'Nothing seems to match.');
     
   });
   
-});
+  it('TC_14.002-08-A |Case insensitive search', () => {
+      cy.get('div[class="login page-header__hyperlinks"] a[class="model-link"]').should('be.visible').click()
+      cy.url().should('include', '/user');
+      cy.get('div:nth-child(3) span:nth-child(1) a:nth-child(1)').click()
```
Comment (Rom-T) [inline cypress/e2e/headerSearchBox.cy.js:80]: I think it's better to use it like this `cy.get('a[href$="/configure"]')`

**Comment by leliks818** [comment]
fix selectors done

**Review by Alzhan-B — APPROVED** [review]
Good!

**Review by Rom-T — APPROVED** [review]
